dev-docs/bidders/colombia.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: Colombia
+description: Prebid Colombia Bidder Adaptor
+biddercode: colombia
+media_types: banner, video, native
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
```

Comment:
This metadata is no longer supported. Please replace with

```
gpp_sids: (list of supported protocols)
```
As noted in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

However -- I don't see your adapter actually reading the GPP string and sending it in your 'payload' object.

So at this point, I think you just need to remove the gpp_supported attribute entirely.

Reply:
Removed